### PR TITLE
feat(core): accept style argument for create-nx-workspace

### PR DESCRIPTION
**Pull request by pesoklp13**
* add style as an available argument for create-nx-workspace command
* updated determineStyle based on parsed argument if passed into command

## Current Behavior (This is the behavior we have today, before the PR is merged)

unable to set stylesheet when creating workspace with preset different from "empty"

## Expected Behavior (This is the new behavior we can expect after the PR is merged)

when valid style option selected when selected other then "empty" preset" it will be applied to new project created alongside with workspace

### CODE REVIEW

File: packages/create-nx-workspace/bin/create-nx-workspace.ts [hunk 1]
```diff
@@ -107,6 +107,9 @@ function showHelp() {
     appName                   the name of the application created by some presets  
 
     cli                       CLI to power the Nx workspace (options: "nx", "angular")
+    
+    style                     style option for determine default style schematics when no empty preset selected 
+                              options: ("css", "scss", "styl", "less") for react also ("styled-components", "@emotion/styled")
```
Comment (jdpearce) [inline packages/create-nx-workspace/bin/create-nx-workspace.ts]: `...for React/Next.js also ("styled-components"...`

File: packages/create-nx-workspace/bin/create-nx-workspace.ts [hunk 2]
```diff
@@ -107,6 +107,9 @@ function showHelp() {
     appName                   the name of the application created by some presets  
 
     cli                       CLI to power the Nx workspace (options: "nx", "angular")
+    
+    style                     style option for determine default style schematics when no empty preset selected
```
Comment (jdpearce) [inline packages/create-nx-workspace/bin/create-nx-workspace.ts]: `default style option to be used when a non-empty preset is selected`

**Review by catfireparty — CHANGES_REQUESTED** [review]
Just some minor text changes, but otherwise this looks fine to me.

How did you test it?

**Comment by pesoklp13** [comment]
@jdpearce cause there is no tests for this purpose I have just builded project and then tested it inside.

In `build/packages/create-nx-workspace/bin` I have started this command as the proof my solution works

`node create-nx-workspace.js pesok-test --preset=angular --appName=pesok-test-app --cli=angular --style=scss`

it created whole structure as I expected and for the `angular.json` file there is also 
```
"pesok-test-app": {
      "projectType": "application",
      "schematics": {
        "@nrwl/angular:component": {
          "style": "scss"
        }
      },
```

File: packages/create-nx-workspace/bin/create-nx-workspace.ts (hunk 1 above)
Comment (pesoklp13) [reply packages/create-nx-workspace/bin/create-nx-workspace.ts]: updated

File: packages/create-nx-workspace/bin/create-nx-workspace.ts (hunk 2 above)
Comment (pesoklp13) [reply packages/create-nx-workspace/bin/create-nx-workspace.ts]: updated

**Review by catfireparty — APPROVED** [review]

**Pull request merged**